.travis.yml

```diff
@@ -1,9 +1,13 @@
 script:
   rake test --trace
 
+before_install:
+  - gem update --system 
+
 rvm:
   - 1.9.2
   - 1.9.3
```

Comment:
What about Ruby 2.0, 2.1, 2.2, 2.3, and 2.4?

Reply:
The ruby I want to use is MRI 2.5 and I don’t care about other versions or implementations. The purpose of this PR is to ensure that I can use MRI 2.5 safely. If you want to use other MRI versions or other implementations, you can PR it. Thanks.